lib/src/components/NxTab/NxTab.scss

```diff
@@ -6,16 +6,51 @@
  */
  @import '../../scss-shared/nx-colors';
  @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+
+.nx-tabs {
+  margin: $nx-spacing-md 0 $nx-spacing-l 0;
+}
 
 .nx-tab {
-  border-bottom: 4px solid transparent;
+  border-bottom: 2px solid transparent;
   cursor: pointer;
   display: inline-block;
   list-style: none;
-  padding: $nx-spacing-xs $nx-spacing-l;
+  margin: 0 $nx-spacing-md -1px 0;
+  // subtract border thickness for consistent spacing
+  padding: $nx-spacing-md $nx-spacing-l ($nx-spacing-md - 2px) $nx-spacing-l;
+  text-align: center;
+  min-width: 76px;
 
   &.active {
-    border-bottom-color: $nx-background-color-interactive;
+    @include semi-bold();
+
+    border-bottom-color: #045dd1;
+    color: #045dd1;
+  }
+
+  &:hover {
+    @include semi-bold();
+  }
+
+  &:focus {
+    border-bottom-width: 1px;
+    box-shadow: 0 0 3px 1px transparentize(#02a6d9, 0.7);
+    outline: solid 1px #02a6d9;
+  }
+
+  a:hover, a:focus {
     font-weight: bold;
+    letter-spacing: 0
+  }
+
+  // a hidden copy of the text, always at semi-bold weight, to enfore consistent element width
+  .hidden-children {
```

Comment:
When I suggested this classname in slack I meant it only as pseudo-code.  For the actual impl I'd say it should be something like `.nx-tab__hidden-children`

Reply:
> When I suggested this classname in slack I meant it only as pseudo-code. For the actual impl I'd say it should be something like `.nx-tab__hidden-children`

I waffled on that, but you're right it should reference `tab` just to make it specific. https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/3a74a316a9f1948537389aed91b2d849dfb6ae8a